front-end-tools/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 ## Unreleased changes
 
+## 2.1.0
+
+- Fix the `isModuleReferenceAlreadyDeployedStep2` check. `isModuleReferenceAlreadyDeployedStep2` uses the module reference from the input field in step 2 while the `isModuleReferenceAlreadyDeployedStep1` uses the module reference from the calculation in step 1.
```

Comment:
The changelog for a tool should typically reflect user understandable or observable changes. 

So I'd avoid mentioning specific function names which have no meaning outside of the codebase.